src/Integration.Vsix/SonarLintTelemetryPackage.cs

```diff
@@ -52,7 +52,7 @@ protected override void Dispose(bool disposing)
                 return;
             }
 
-            this.telemetryManager?.Dispose();
+            (this.telemetryManager as IDisposable)?.Dispose();
```

Comment:
This looks odd. Why is `ITelemetryManager` no longer IDisposable?

Reply:
Because as pointed out by @duncanp-sonar on some other commit the pattern mainly used in SLVS is to not have `IDisposable` at the interface level.